client/shipping-label/state/selectors/errors.js

```diff
@@ -58,7 +58,22 @@ const getPackagesErrors = ( values ) => _.mapValues( values, ( pckg ) => {
 	return errors;
 } );
 
-const getRatesErrors = ( values ) => _.mapValues( values, ( ( rate ) => rate ? null : __( 'Please choose a rate' ) ) );
+const getRatesErrors = ( { values: selectedRates, available: allRates } ) => {
+	return {
+		server: _.mapValues( allRates, ( rate ) => {
+			if ( rate.errors ) {
+				return rate.errors.map( ( error ) => {
+					return {
+						message: ( error.userMessage ||
+							error.message ||
+							"We couldn't get a rate for this package, please try again." ),
+					};
```

Comment:
You're right, should just be a plain error string. Fixed in 189f98ef79af32d59791a90ae8ae9b0afa6447da